package.json

```diff
@@ -33,6 +33,7 @@
     "tauri:dev": "lerna run --parallel --scope '@padloc/{server,tauri}' --parallel dev",
     "repl": "cd packages/server && npm run repl && cd ../..",
     "test": "lerna run test",
-    "locale:extract": "lerna run extract --scope '@padloc/locale'"
+    "locale:extract": "lerna run extract --scope '@padloc/locale'",
+    "add": "lerna add"
```

Comment:
We don't need to have this, but made it easier for me to add a new dependency (`npm run add -- 1pux-to-csv --scope=@padloc/app` vs `./node_modules/.bin/lerna add 1pux-to-csv --scope=@padloc/app`).

Reply:
Almost seems like lerna should be replaced with [`npm workspaces`](https://docs.npmjs.com/cli/v7/using-npm/workspaces). I could add a command to do this, but would need to be in something like a `Makefile` or `.sh` script. If that doesn't sound like a terrible idea, I can implement it in a separate PR.

Reply:
Let's put a pin in it for now - the pain definitely isn't big enough to warrant spending too much time on this right now.